packages/viewer/src/rendering/__tests__/coordinates.spec.ts

```diff
@@ -1,125 +1,41 @@
-import { Dimensions, Point } from '@vertexvis/geometry';
-import {
-  computeNormalizedDeviceCoordinates,
-  computeWorldPosition,
-} from '../coordinates';
-import {
-  inverseProjectionMatrix,
-  inverseViewMatrix,
-} from '../../rendering/matrices';
+jest.mock('@vertexvis/stream-api');
 
-describe(computeNormalizedDeviceCoordinates, () => {
-  const viewport = Dimensions.create(100, 100);
-  const point1 = Point.create(50, 50);
-  const depth1 = 0;
-  const point2 = Point.create(0, 0);
-  const depth2 = 0.5;
-  const point3 = Point.create(100, 100);
-  const depth3 = 1;
-
-  it('returns correct NDC values', () => {
-    expect(
-      computeNormalizedDeviceCoordinates(viewport, point1, 1, depth1)
-    ).toMatchObject({
-      x: 0,
-      y: 0,
-      z: -1,
-    });
-
-    expect(
-      computeNormalizedDeviceCoordinates(viewport, point2, 1, depth1)
-    ).toMatchObject({
-      x: -1,
-      y: 1,
-      z: -1,
-    });
-
-    expect(
-      computeNormalizedDeviceCoordinates(viewport, point3, 1, depth1)
-    ).toMatchObject({
-      x: 1,
-      y: -1,
-      z: -1,
-    });
-
-    expect(
-      computeNormalizedDeviceCoordinates(viewport, point1, 1, depth2)
-    ).toMatchObject({
-      x: 0,
-      y: 0,
-      z: 0,
-    });
-
-    expect(
-      computeNormalizedDeviceCoordinates(viewport, point1, 1, depth3)
-    ).toMatchObject({
-      x: 0,
-      y: 0,
-      z: 1,
-    });
-  });
-});
+import { BoundingBox, Dimensions, Point, Vector3 } from '@vertexvis/geometry';
+import { computeWorldPosition } from '../coordinates';
+import { Camera } from '../../scenes';
+import { StreamApi } from '@vertexvis/stream-api';
 
 describe(computeWorldPosition, () => {
-  const near = 5;
-  const far = 15;
   const viewport = Dimensions.create(100, 100);
-  const inverseProjection = inverseProjectionMatrix(near, far, 45, 1);
-  const inverseView = inverseViewMatrix({
-    position: { x: 0, y: 0, z: 5 },
-    lookAt: { x: 0, y: 0, z: 0 },
-    up: { x: 0, y: 1, z: 0 },
-  });
+  const camera = new Camera(
+    new StreamApi(),
+    1,
+    {
+      position: { x: 0, y: 0, z: 5 },
+      lookAt: { x: 0, y: 0, z: 0 },
+      up: { x: 0, y: 1, z: 0 },
+    },
+    BoundingBox.create(Vector3.origin(), { x: 0, y: 0, z: 100 })
+  );
   const point = Point.create(50, 50);
 
   it('should return correct world position for near-plane depth', () => {
     const depth = 0;
 
-    expect(
-      computeWorldPosition(
-        inverseProjection,
-        inverseView,
-        viewport,
-        point,
-        depth,
-        near,
-        far,
-        1
-      ).z
-    ).toBe(0);
+    console.log(camera.near, camera.far);
```

Comment:
Should remove.

Reply:
Missed that one, removed in https://github.com/Vertexvis/vertex-web-sdk/pull/175/commits/263d4f616a453378aa2525d63a186a39a0b6e7d3